### ISSUE TITLE
Does this work on m1/m2 mac

### ISSUE THREAD

**Issue by rootux** [moostrik/ofxFlowTools#34]
Anyone tried running this on an m1?

**Comment by artificiel** [comment]
yes, using nightly, but there is a reference to `std::bind2nd` which has been removed from C++ — I don't see a quick fix so I just commented out the line not sure what function it had. also some return types have to be fixed to `void` on a couple of setters as the compiler disallows non-returns. there 2 things should be reported as issues.

<img width="2439" alt="image" src="https://github.com/moostrik/ofxFlowTools/assets/2172370/ca9e506b-5a47-481e-a4d9-2f02b373df93">

**Comment by stephanschulz** [comment]
try replacing `std::transform(_v.begin(), _v.end(), diff.begin(), std::bind2nd(std::minus<float>(), mean));`

with ` std::transform(_v.begin(), _v.end(), diff.begin(),[mean](float x) { return x - mean; });`

for me this prevents the compile error. but I am not sure how to test if it still performs the same task.
